CMakeLists.txt

```diff
@@ -251,6 +257,11 @@ option(ONNX_ML "Enable traditional ONNX ML API." ON)
 option(HAVE_SOVERSION "Whether to add SOVERSION to the shared objects" OFF)
 option(USE_DEPLOY "Enable torch::deploy embedded python interpreter" OFF)
 
+if($ENV{BUILD_SPLIT_CUDA})
```

Comment:
I'm not sure this is 100% necessary since USE_CPP_COVERAGE seems to work without additional code.

Reply:
There are no need, as setup.py should do that for you

Reply:
Ah thanks, this would explain why when I just ran `cmake -DBUILD_SPLIT_CUDA=ON ..` from within the build folder, it did not get set properly.